src/serlio/materials/MaterialUtils.cpp

```diff
@@ -35,6 +35,21 @@ MObject findNamedObject(const std::wstring& name, MFn::Type fnType) {
 	return MObject::kNullObj;
 }
 
+adsk::Data::Structure* createNewMaterialInfoMapStructure() {
+	adsk::Data::Structure* fStructure;
```

Comment:
consider moving the declaration to line 42

Reply:
done (did the same for the other fstructure in MayaCallbacks)